### DISCUSSION TITLE
set default CACHE_AUTO_PURGE to true

### DISCUSSION THREAD

**Discussion by ctholho** [directus/directus#12129] [Feature Requests]
In the spirit of the principle of least astonishment, I would propose to change the default value of `CACHE_AUTO_PURGE` to `true`. To soften the blow for anyone who relies on this to be false and does not set CACHE_AUTO_PURGE explicitly to false, we could add a nice deprecation warning for any deployments that have `CACHE_ENABLED === true` and `typeof CACHE_AUTO_PURGE === 'undefined'`.

**Comment by rijkvanzanten** [comment]
Agreed! Wanna open a PR for that?

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
